config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   root to: "wishlists#index"
-  resources :wishlists, only: [:index, :new, :create]
+
+  resources :wishlists, only: [:index, :new, :create, :show] do
+    resources :items, only: [:show]
```

Comment:
You are correct, I did this by mistake since at the beginning I wanted to code Items actions in the same PR.